package.json

```diff
@@ -17,13 +17,13 @@
     "clean-client": "rm -rf _inc/build/*.js _inc/build/*.css _inc/build/*.map _inc/build/*.html",
     "distclean": "rm -rf node_modules && yarn cache clean",
     "build": "yarn && yarn build-client",
-    "build-client": "./node_modules/.bin/gulp",
-    "build-production": "yarn clean-client && yarn build-languages && ./node_modules/.bin/gulp languages:extract && NODE_ENV=production BABEL_ENV=production yarn build",
-    "build-languages": "./node_modules/.bin/gulp languages",
+    "build-client": "gulp",
```

Comment:
Unrelated with this issue, but I had `yarn build-client` failing for me (Windows 10) because `./node_modules/...` is not valid CMD syntax. On the context of NPM scripts, that full path is redundant, since all executables from `./node_modules/.bin/*` are included to the `$PATH` automatically.